code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -95,8 +111,33 @@
 /mob/living/silicon/pai/can_buckle()
 	return FALSE
 
+/mob/living/silicon/pai/handle_atom_del(atom/A)
+	if(A == hacking_cable)
```

Comment:
Couldn't this all be a switch()?

Reply:
don't think so, switch requires constant expression